src/components/forms/VpnServerForm.js

```diff
@@ -0,0 +1,386 @@
+            labelText="Enable Split Tunneling"
+            defaultToggled={this.state.enable_split_tunneling}
+            onToggle={() => this.handleToggle("enable_split_tunneling")}
+            id="vpn-server-enable-split-tunneling"
+            className="fieldWidthSmaller"
+          />
+          {/* client_idle_timeout input */}
+          <NumberInput
+            placeholder="600"
+            label="Client Idle Timeout (sec)"
+            id="vpn-serrver-client-idle-timeout-seconds"
+            allowEmpty={true}
+            value={this.state.client_idle_timeout || ""}
+            step={1}
+            onChange={this.handleNumberInputChange}
+            name="client_idle_timeout"
+            hideSteppers={true}
+            min={0}
+            max={28800}
+            invalid={isRangeInvalid(this.state.client_idle_timeout, 0, 28800)}
+            invalidText="Must be a whole number between 0 and 28800."
+            className="fieldWidth leftTextAlign"
+          />
+          {/* port input */}
+          <NumberInput
+            label="Port"
+            id="vpn-serrver-port"
+            allowEmpty={true}
+            value={this.state.port || ""}
+            step={1}
+            onChange={this.handleNumberInputChange}
+            name="port"
+            hideSteppers={true}
+            min={1}
+            max={65535}
+            invalid={isRangeInvalid(this.state.client_idle_timeout, 1, 65535)}
+            invalidText="Must be a whole number between 1 and 65535."
+            className="fieldWidth leftTextAlign"
+          />
+          {/* protocol */}
+          <IcseSelect
+            formName="vpn-server-protocol"
+            groups={["TCP", "UDP"]}
+            value={this.state.protocol.toUpperCase()}
+            labelText="Protocol"
+            name="protocol"
+            handleInputChange={(event) =>
+              this.handleInput("protocol", event, true)
+            }
+            className="fieldWidthSmaller"
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          {/* resource group */}
+          <IcseSelect
+            formName="vsi_form"
+            name="resource_group"
+            className={classNameModalCheck}
+            labelText="Resource Group"
+            groups={this.props.resourceGroups}
+            value={this.state.resource_group}
+            handleInputChange={this.handleInputChange}
+          />
+          {/* vpc */}
+          <IcseSelect
+            formName="vpn_server"
+            name="vpc"
+            className={classNameModalCheck}
+            labelText="VPC"
+            groups={this.props.vpcList}
+            value={this.state.vpc}
+            handleInputChange={this.handleInputChange}
+            invalid={checkNullorEmptyString(this.state.vpc)}
+            invalidText="Select a VPC."
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          {/* subnets and security groups */}
+          <SubnetMultiSelect
```

Comment:
let's change this to a dropdown instead of multiselect

Reply:
@jvallexm shall I do the same for the security groups as well?